### PR TITLE
Project spa navigation

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes

Added project navigation and page layout

## Related issue

N/A

## Checklist

Things for the developer to confirm they've done before the PR should be accepted:

*   [ ] ~CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~
*   [ ] ~Tests added (or description of how to test) for any new features.~
*   [ ] ~User documentation updated for UI or technical changes.~

### CODE REVIEW

File: src/main/webapp/resources/js/layouts/project-layout.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  const { data: details = {} } = useGetProjectDetailsQuery(projectId);
+
+  return (
+    <PageHeader
+      title={details.label}
+      subTitle={details.description}
+      style={{ margin: `0 25px` }}
+    >
+      <div style={{ backgroundColor: `#ffffff` }}>
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/layouts/project-layout.tsx:22]: Get variable from app.css for the background color

File: src/main/webapp/resources/js/redux/endpoints/project.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { api } from "./api";
+import { TAG_PROJECT } from "./tags";
+
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/redux/endpoints/project.ts:3]: File comment

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
Looks good so far. Just a few small things below. Other than that I just need to test it out

File: src/main/webapp/resources/js/layouts/project-layout.tsx (hunk 1 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/layouts/project-layout.tsx:22]: Can't, app.css is not loaded onto this page. Still thinking about how to handle this.

File: src/main/webapp/resources/js/redux/endpoints/project.ts (hunk 2 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/redux/endpoints/project.ts:3]: Updated in 5ef5821

File: src/main/webapp/resources/js/layouts/project-layout.tsx (hunk 1 above)
Comment (deepsidhu85) [reply src/main/webapp/resources/js/layouts/project-layout.tsx:22]: Sounds good. We can handle that in a later pr

**Review by deepsidhu85 — APPROVED** [review]

**Pull request merged**